Integrations/SymantecMSS/SymantecMSS.yml

```diff
@@ -1,99 +1,151 @@
+category: Case Management
```

Comment:
What was changed in this file? Seems like a lot just changed it’s place

Reply:
Thats exactly what happened, the PyCharm plugin did the manipulations, but no change was done.